src/openPr.js

```diff
@@ -93,6 +95,25 @@ module.exports = async function ({ context, inputs, packageVersion }) {
       },
       inputs
     )
+
+    // attach artifact
+    const buildDir = inputs['release-artifact-build-folder']
```

Comment:
I'm not sure this is the right place where to do this. The try/catch block where you positioned this code is for creating the PR, it doesn't look right to nest this code in there